controllers/clustergroupupgrade_controller.go

```diff
@@ -1018,33 +1102,21 @@ func (r *ClusterGroupUpgradeReconciler) getPolicyContent(
 	for _, template := range policyTemplatesArr {
 		// Get to the metadata name of the ConfigurationPolicy.
 		objectDefinition := template.(map[string]interface{})["objectDefinition"]
-		if objectDefinition == nil {
```

Comment:
I see that we currently only call this with already validated policies. Maybe add some comment for the function so that it doesn't get reused with newly loaded policies that are not validated yet.

Reply:
Sure, will update

Reply:
done